src/components/application_manager/include/application_manager/commands/hmi/request_from_hmi.h

```diff
@@ -63,9 +63,28 @@ class RequestFromHMI : public CommandImpl, public event_engine::EventObserver {
                     const hmi_apis::FunctionID::eType function_id,
                     const hmi_apis::Common_Result::eType result_code);
 
-  void FillCommonParametersOfSO(smart_objects::SmartObject* message,
-                                uint32_t correlation_id,
-                                hmi_apis::FunctionID::eType function_id);
+  /**
+   * @brief SendResponse allows to send error response to hmi
+   * @param correlation_id the correlation id for the response.
+   * @param function_id the function id for which response will be sent
+   * @param result_code the result code.
+   */
+  void SendErrorResponse(const uint32_t correlation_id,
+                         const hmi_apis::FunctionID::eType function_id,
+                         const hmi_apis::Common_Result::eType result_code,
+                         std::string error_message);
+
+ private:
+  /**
+   * @brief Fills common parameters for SO
+   * @param Contains SO for filling
+   * @param correlation_id the correlation id for the response.
+   * @param function_id the function id for which response will be sent
+   */
+  void FillCommonParametersOfSO(
+      NsSmartDeviceLink::NsSmartObjects::SmartObjectSPtr& message,
```

Comment:
@VVeremjova Why did you change namespace smart_objects to NsSmartDeviceLink::NsSmartObjects. I think name space "smart_objects" is better.